### ISSUE TITLE
Should Optix have its own resoucre variable similar to ogre?

### ISSUE THREAD

**Issue by Levi-Armstrong** [ignitionrobotics/ign-rendering#111]
Currently the Ogre and Ogre2 plugin leverage a separate environment variable/compiler definition to locate its resources (OGRE_RESOURCE_PATH and OGRE2_RESOURCE_PATH).  In the case Optix it is using the IGN_RENDERING_RESOURCE_PATH; should it have its own variable for locating Optix specific resources similar to how it is done in the Ogre and Ogre2 plugin?

https://github.com/ignitionrobotics/ign-rendering/blob/bdf08866775e7a3f0953f7a2b802577199247242/optix/src/OptixRenderEngine.cc#L83-L87

**Comment by iche033** [comment]
yeah that's a good idea. We should still keep the `IGN_RENDERING_RESOURCE_PATH` there for backward compatibility. 

Removing this from Core Development since optix is lower priority now.
